r/src/get_metadata_information.r

```diff
@@ -3,21 +3,24 @@
 # The doublet scores were computing in the data-ingest script. To compute then, we
 # use the package Scrublet [1]
 getDoubletScore <- function(req) {
-     
-    # Check if the experiment has doublet_scores stored in rds file
-    if(!"doublet_scores"%in%colnames(data@meta.data)){
-        stop("Doublet scores are not computed for this experiment.")
-    }
-
-    # Subset the doublet_scores ordering by cells_id
-    result <- data@meta.data[order(data$cells_id, decreasing = F), "doublet_scores"]
-
-    # Be aware of possible na values
-    if(any(is.na(result)))
-        warning("There are missing values in the doublet_scores results")
-
-    return(result)
-
+  # Check if the experiment has doublet_scores stored in rds file
+  if(!"doublet_scores"%in%colnames(data@meta.data)){
+    stop("Doublet scores are not computed for this experiment.")
+  }
+  
+  # Subset the doublet_scores ordering by cells_id
+  result <- data@meta.data[order(data$cells_id, decreasing = F), "doublet_scores"]
+  result<- as.data.frame(result)
+  result$cells_id <- data@meta.data$cells_id
```

Comment:
```suggestion
  result <- data@meta.data[order(data$cells_id, decreasing = F), c("doublet_scores", "cells_id")]
  result <- as.data.frame(result)
```

Reply:
The above code could prevent that for example if the initial data is not ordered, we can have mismatched the `doublet_scores` and the `cells_id`; since in line 12 we order but not in line 14.